ParkingSpace.Web/Views/GateIn/Index.cshtml

```diff
@@ -6,7 +6,7 @@
     (ParkingTicket)TempData["newTicket"];
 }
 
-<h2>Gate In</h2>
+<h2>Gate In [@ViewBag.GateId]</h2>
```

Comment:
I think, display gate number in another line is better, right? :smirk: 

For example:

<h2>Gate In</h2>
Gate: 08